docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -153,24 +153,24 @@ In this section you will create your first Lieutenant configuration objects usin
 export LIEUTENANT_TOKEN=$(kubectl -n lieutenant get secret $(kubectl -n lieutenant get sa api-access-synkickstart -o go-template='{{(index .secrets 0).name}}') -o go-template='{{.data.token | base64decode}}')
 export LIEUTENANT_AUTH="Authorization: Bearer ${LIEUTENANT_TOKEN}"
 export LIEUTENANT_URL="lieutenant.${K3S_INGRESS_IP}.nip.io"
-export GITLABCOM_USERID="MYUSER"
+export GITLABCOM_USERNAME="MYUSER"
 ----
 
 . Create a *Lieutenant Tenant* via the API
 +
 [source,shell]
 ----
-TENANT_ID=$(curl -s -H $LIEUTENANT_AUTH -H "Content-Type: application/json" -X POST --data "{\"displayName\":\"My first Tenant\",\"gitRepo\":{\"url\":\"ssh://git@gitlab.com/${GITLABCOM_USERID}/mytenant.git\"}}" http://${LIEUTENANT_URL}/tenants | jq -r ".id")
+TENANT_ID=$(curl -s -H "$LIEUTENANT_AUTH" -H "Content-Type: application/json" -X POST --data "{\"displayName\":\"My first Tenant\",\"gitRepo\":{\"url\":\"ssh://git@gitlab.com/${GITLABCOM_USERNAME}/mytenant.git\"}}" http://${LIEUTENANT_URL}/tenants | jq -r ".id")
```

Comment:
```suggestion
TENANT_ID=$(curl -s -H "$LIEUTENANT_AUTH" -H "Content-Type: application/json" -X POST --data "{\"displayName\":\"My first Tenant\",\"gitRepo\":{\"url\":\"ssh://git@gitlab.com/${GITLABCOM_USERNAME}/mytenant.git\"}}" "http://${LIEUTENANT_URL}/tenants" | jq -r ".id")
```

While we are at it. Wrapping URLs in quotes prevent auto escaping on some terminals (namely the one of macOS).